saltstack/base/salt/prerequisites/init.sls

```diff
@@ -13,6 +13,8 @@ include:
   - {{ slspath }}.cert-tool
   - {{ slspath }}.dnsmasq
   - {{ slspath }}.user
+  - {{ slspath }}.selinux
```

Comment:
This will not work on docker build. Shouldn't we put this into else part of the docker check at line 6?